doc/modules/impute.rst

```diff
@@ -105,9 +105,12 @@ imputation round are returned.
 
 .. note::
 
-   This estimator is still **experimental** for now: the predictions
-   and the API might change without any deprecation cycle. To use it,
-   you need to explicitly import ``enable_iterative_imputer``.
+   This estimator is still **experimental** for now: default parameters or
+   details of behaviour might change without any deprecation cycle,
+   specifically convergence criteria (`issue 14338 <https://github.com/scikit-learn/scikit-learn/issues/14338>`_),
```

Comment:
```suggestion
   specifically convergence criteria (:issue:`14338`),
```
You can use these shortcuts from the Sphinx-issues extension

Reply:
TIL. Done.